at-chuongphu/excercise6.rb

```diff
@@ -0,0 +1,32 @@
+# ex 1
+cong = Proc.new{ |*ar| ar.inject{|a, b| a+b} }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/eb9d75251b04dc277a5bfb9ccd46566701b31196/at-chuongphu/excercise6.rb#L2"></span>Style/Proc: Use <code>proc</code> instead of <code>Proc.new</code>. (https://github.com/rubocop-hq/ruby-style-guide#proc)</td>
    </tr>
  </tbody>
</table>